freyja_controller.launch

```diff
@@ -39,43 +38,42 @@
     <param name="vicon_topic" type="string" value="$(arg vicon_topic)" />
   </node>
 
-	<!-- an example trajectory node for reference -->
-  <node name="trajectory_provider" pkg="freyja_trajectory_provider" type="trajectory_provider_node"
-				if="$(eval not use_external_trajectory)">
-		<!-- "hover", "circle", "hover2" -->
-  	<param name="example_traj_type" type="string" value="hover"/>
-	</node>
-  
   <!-- control node (see velctrl flag above) -->
   <node name="lqg_controller" pkg="lqg_control" type="lqg_control_node"
-  			output="screen" if="$(eval not use_velctrl_only)">
-		<param name="controller_rate" value="45" type="int" />
-  	<param name="total_mass" type="double" value="$(arg total_mass)"/>
-  	<param name="use_stricter_gains" type="bool" value="false" />
-  	<param name="estimator_rate" type="int" value="70" />
-  	<param name="bias_compensation" type="bool" value="$(arg bias_compensation)" />
+      output="screen" if="$(eval not use_velctrl_only)">
+    <param name="controller_rate" value="45" type="int" />
+    <param name="total_mass" type="double" value="$(arg total_mass)"/>
+    <param name="use_stricter_gains" type="bool" value="false" />
+    <param name="estimator_rate" type="int" value="70" />
+    <param name="bias_compensation" type="bool" value="$(arg bias_compensation)" />
   </node>
   
   <node name="lqr_vel_controller" pkg="lqr_control" type="lqr_vel_ctrl_node"
-  			if="$(eval use_velctrl_only)">
+        if="$(eval use_velctrl_only)">
     <param name="controller_rate" type="int" value="30" />
     <param name="total_mass" value="$(arg total_mass)" type="double" />
   </node>
 
   <!-- autopilot communication node -->
   <node name="asctec_handler" pkg="asctec_handler" type="asctec_handler_node"
-  			if="$(eval autopilot=='asctec')">
+        if="$(eval autopilot=='asctec')">
     <param name="serial_port" value="/dev/ttyUSB0" />
   </node>
   
   <!-- apm/px4 needs a handler for translation, plus a mavros node -->
   <group if="$(eval autopilot=='arducopter')">
-  	<node name="apm_handler" pkg="apm_handler" type="apm_handler_node" />
-  	<include file="$(find freyja_configfiles)/apm_nimbus.launch">
-  		<arg name="fcu_url" value="$(arg comm_port)"/>
-  	</include>
+    <node name="apm_handler" pkg="apm_handler" type="apm_handler_node" />
+    <include file="$(find freyja_configfiles)/apm_nimbus.launch">
+      <arg name="fcu_url" value="$(arg comm_port)"/>
+    </include>
   </group>
+
+  <node name="waypoint_manager" pkg="waypoint_manager" type="waypoint_manager_node" output="screen"
+				if="$(eval use_waypoint_handler)"> 
+	</node>
+
+  <include file="$(find freyja_examples)/launch/example.launch" if="$(eval use_examples)" >
+    <arg name="example_number" value="$(arg example_number)"/>
```

Comment:
Don't see `example_number` as an argument ..

Reply:
Sorry, this must have been removed in the merge. Fixed